app/activeplan/homepage/get.controller.test.js

```diff
@@ -0,0 +1,43 @@
+const controller = require('./get.controller')
+
+describe('showHomepage', () => {
+  const req = {
+    params: {
+      planId: 12,
+      id: 123,
+    },
+    session: {
+      'x-auth-token': '1234',
+      planStarted: false,
+    },
+  }
+  const res = {
+    render: jest.fn(),
+  }
+
+  afterEach(() => {
+    req.session.planStarted = false
```

Comment:
this really only applies to one test; we manually set it for the other. Maybe pull out the default value and the cleanup and stick the set line in the pertinent test?